dashboard/test/ui/features/course_versions.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Course versions
+
+@as_student
+@no_mobile
+Scenario: Version warning announcement on course and unit overview pages
+  # course and unit pages do not show version warning initially
+
+  When I am on "http://studio.code.org/courses/csp-2017?enableExperiments=courseVersions"
```

Comment:
Do we have to turn off experiments at the end of these tests or are they just forgotten since we use SauceLabs?

Reply:
Great question! After each scenario, it turns out we either clear `sessionStorage` (on iOS), or get an entirely new browser. The implementation of this can be seen by grepping for `slow_browser?` in https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/support/connect.rb

This particular test is tagged with `@no_mobile`, so it should still work fine, however because experiments are stored in `localStorage` (not `sessionStorage`) it's possible the wrong thing could happen when enabling an experiment in a Scenario which runs on iOS.

I'll make a follow-up PR to add a call to localStorage.clear() to connect.rb.